Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -417,42 +417,49 @@ def lift : MultilinearMap R s E ≃ₗ[R] (⨂[R] i, s i) →ₗ[R] E where
 
 variable {φ : MultilinearMap R s E}
 
+-- Why is it necessary to replace `lift` with `lift.toLinearMap` here?
 @[simp]
-theorem lift.tprod (f : ∀ i, s i) : lift φ (tprod R f) = φ f :=
+theorem lift.tprod (f : ∀ i, s i) : lift.toLinearMap φ (tprod R f) = φ f :=
```

Comment:
Hello @Vierkantor , I tried with this instance that you indicated me (it took me time to understand where to insert it); it works fine with it. But since it was found automatically, I removed it, and it still works…